### DISCUSSION TITLE
Le bouclier de brave bloque TAC

### DISCUSSION THREAD

**Discussion by lchk609** [AmauriC/tarteaucitron.js#1192] [General]
Bonjour Amauri,

Je viens de remarquer sur le site sur lequel je travaille que lorsque le shield de brave est activé, le fichier js de TAC ne se chargeait pas et donc engendrait des erreurs de javascript dans la console. A noter que cette erreur survient uniquement sur Brave. 
![image](https://github.com/AmauriC/tarteaucitron.js/assets/148492092/8dc1afd1-5f90-4217-921b-e1f3d28b8c68)

Y a t-il une méthode pour ne pas se faire bloquer par Brave ? 

Bonne journée

**Comment by AmauriC** [comment]
Bonjour,

C'est une fonctionnalité de Brave de bloquer certains scripts.

Pas de solution mis à part utiliser un autre navigateur ; )
